content/en/docs/setup/best-practices/enforcing-pod-security-standards.md

```diff
@@ -75,4 +75,4 @@ ecosystem:
 The decision to go with a _built-in_ solution (e.g. PodSecurity admission controller) versus a
 third-party tool is entirely dependent on your own situation. When evaluating any solution,
 trust of your supply chain is crucial. Ultimately, using _any_ of the aforementioned approaches
-will be better than doing nothing.
\ No newline at end of file
+will be better than doing nothing.
```

Comment:
I'd prefer to omit this change, even if it was an accident. This PR should only touch the Hindi localization.

Reply:
that was an unneccesary commit from my side ,Thanks @sftim , My final version is ready for a review